pkg/kubectl/resource/builder.go

```diff
@@ -60,6 +60,7 @@ type Builder struct {
 
 	singleResourceType bool
 	continueOnError    bool
+	skipFetchingNames  bool
```

Comment:
Also, that's not the least confusing comment I've seen, notwithstanding anything to the contrary.